helper/kubeone-tool-container/Dockerfile

```diff
@@ -43,7 +43,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends\
         locate \
         netcat \
         dnsutils\
-        wget \
```

Comment:
I did not remove it.. I had explicitly added it because I had added `--no-install-recommends`. But with recommeded installs.. wget gets installed automatically so removed that from subsequent commits again.